androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/navigation/AppNavigation.kt

```diff
@@ -18,7 +19,23 @@ fun AppNavHost(
         modifier = modifier
     ) {
         composable(AppDestination.Splash) {
-            SplashScreen()
+            SplashScreen(
+                onLoginSuccess = {
+                    navController.navigate(
+                        route = AppDestination.Home.route,
+                        navOptions = navOptions {
+                            popUpTo(route = AppDestination.Splash.route) {
+                                inclusive = true
+                            }
+                            launchSingleTop = true
+                        }
+                    )
+                }
+            )
+        }
```

Comment:
@kaungkhantsoe we cannot keep `AppDestination` inside the VM and observe it from the screen since `AppDestination` has platform dependencies that cannot be satisfied because the VM is in the `:shared` module.

For instrumentation tests, we can test `AppNavHost` in isolation to test the destinations. [Here's](https://github.com/googlecodelabs/android-compose-codelabs/blob/end/NavigationCodelab/app/src/androidTest/java/com/example/compose/rally/NavigationTest.kt) an example of this approach.